xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm

```diff
@@ -179,26 +179,28 @@
   #end
   #template("hierarchy.vm")
   <div class="row">
-    #if ($displayContentMenu)
-      <div class="col-xs-12 col-md-5 pull-right">
-        #template("menus_content.vm")
-      </div>
-    #end
-    #if (($editor == 'wiki' || $editor == 'wysiwyg') && !$request.section)
-      <div id="editMeta" class="col-xs-12#if ($displayContentMenu) col-md-7#end">
-        #template('editmeta.vm')
-      </div>
-    #else
-      <div id="document-title" class="col-xs-12#if ($displayContentMenu) col-md-7#end"><h1>
-      #if($editor == 'rights')
-        $services.localization.render('core.editors.rights.title', [$escapetool.xml($doc.plainTitle), $doc.getURL()])
-      #elseif($editor == 'object')
-        $services.localization.render('core.editors.object.title', [$escapetool.xml($doc.plainTitle), $doc.getURL()])
-      #elseif($editor == 'class')
-        $services.localization.render('core.editors.class.title', [$escapetool.xml($doc.fullName), $doc.getURL()])
+    <dl>
+      #if ($displayContentMenu)
+        <div class="col-xs-12 col-md-5 pull-right">
+          #template("menus_content.vm")
+        </div>
       #end
-      </h1></div>
-    #end
+      #if (($editor == 'wiki' || $editor == 'wysiwyg') && !$request.section)
+        <div id="editMeta" class="col-xs-12#if ($displayContentMenu) col-md-7#end">
+          #template('editmeta.vm')
+        </div>
+      #else
+        <div id="document-title" class="col-xs-12#if ($displayContentMenu) col-md-7#end"><h1>
+        #if($editor == 'rights')
```

Comment:
Same, there's no ``dt`` or ``dd`` wrapped here, but a level 1 heading.

Reply:
Where do you see on the [Vertical Form](https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/VerticalForms/) documentation an example like this:

```
<dl>
  <div>
    <h1>...</h1>
```

Again, HTML validity is beyond the Vertical Form documentation, which is specific to XWiki. You cannot have vertical forms with invalid HTML.

Reply:
Um, yes that's true. So I can not have `dl` in main file `edit.vm` , that's what I learned.